src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/rope_fusion.hpp

```diff
@@ -4,57 +4,57 @@
 
 #pragma once
 
-#include <ngraph/pass/graph_rewrite.hpp>
+#include <openvino/pass/graph_rewrite.hpp>
```

Comment:
```suggestion
#include "openvino/pass/graph_rewrite.hpp"
```
Because is modified so consider update it to use `""`.
Apply to all similar modified includes.